projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/profile-tag.e2e-spec.ts

```diff
@@ -0,0 +1,44 @@
+import { clickAllowAllFromBanner } from '../../../helpers/anonymous-consents';
+import { Navigation } from '../../../helpers/navigation';
+import { CdsHelper } from '../../../helpers/vendor/cds/cds';
+import * as MercahndisingCarousel from '../../../helpers/vendor/cds/merchandising-carousel';
+import { ProfileTagHelper } from '../../../helpers/vendor/cds/profile-tag';
+describe('Profile-tag component', () => {
+  beforeEach(() => {
+    cy.server();
+    CdsHelper.setUpMocks();
+  });
+  it('Should wait for a user to accept consent before sending an event', () => {
+    Navigation.visitHomePage();
+    cy.wait(1000);
```

Comment:
is there a reason to explicitly wait? We are trying to avoid this situation because we don't know if it `x` thing will show up after this.
- From the looks of it, you are waiting for homepage to load. 
- you can import `waitForPage` from `checkout-flow.ts` to wait for the page to load
-- for example, 
---- const homePage = checkout.waitForPage('homepage', 'getHomePage');
----  Navigation.visitHomePage();
---- cy.wait(`@$homepage}`);

Reply:
Here is the tests case I was trying to accomplish here:
- a event should not be sent before the consent banner has been clicked

Although there is no way to guarantee that 1s is long enough to wait in order to make sure something doesn't happen, in practice it should be plenty long. 
Is there another way to test this, or should this be removed?

Reply:
another way is to use what the cms pages. In case as above, where I said to use homepage, as it will go to that page.